### PR TITLE
Upgrades turbo to 2.0.6

### PR DESCRIPTION
**Pull request by bruugey**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [Name of ticket](https://jira.mongodb.org/browse/[name-of-ticket])

### For bug fixes, new features & breaking changes

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

### For new components

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README
- [ ] I have verified the Live Example looks as intended on the design website.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,5 +79,6 @@
   },
   "bugs": {
     "url": "https://github.com/mongodb/leafygreen-ui/issues"
-  }
+  },
+  "packageManager": "yarn@1.22.19"
```
Comment (TheSonOfThomp) [inline package.json:83]: why this version? `1.22.22` is the latest

**Comment by TheSonOfThomp** [comment]
Previously running `yarn clean` (or more specifically, `yarn clean:modules`) would clear the turbo cache. Should we create a new script to restore this functionality?

File: package.json (hunk 1 above)
Comment (bruugey) [reply package.json:83]: i think thats whats on my OS lol - the codemod added this

Comment (TheSonOfThomp) [reply package.json:83]: Lets add a `^` to the version number to be safe

**Review by TheSonOfThomp — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (bruugey) [reply package.json:84]: that syntax is not allowed

**Comment by bruugey** [comment]
> Previously running `yarn clean` (or more specifically, `yarn clean:modules`) would clear the turbo cache. Should we create a new script to restore this functionality?

Added a clean:cache script, lmk if that's what you were looking for

**Pull request merged**